docker/docker-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+# Currently supported values for ${TARGET} are 'bionic' or 'trusty'
+# pwd should be the repos root.
+#
+# Example:
+#
+# /home/user/git/fritzing-app:$ TARGET=trusty ./docker/docker-build.sh
+#
+# Step 1/7 : FROM registry.gitlab.com/fritzing/fritzing-app_playground/build:trusty
+# ---> 2c4428f29501
+# ....
+
+docker build -t fritzing_${TARGET} docker/fritzing_${TARGET} --build-arg user_id=`id -u`
```

Comment:
What are you trying to achieve?

Why do you need so many Docker images for just 2 builds?

Reply:
I separated the Dockerfiles so that we can store a docker image at a registry, but still run the build with custom user permissions on the local file system. Otherwise docker would store all the object files as root.